frontend/src/components/NotificationsBar/Notification/Notification.tsx

```diff
@@ -12,16 +12,24 @@ interface NotificationProps {
 
 const NotificationPlate: React.FC<NotificationProps> = ({ note }) => {
   const isOldNotification = note.status === NOTIFICATION_STATUSES.old;
+  const isUserNotifications = note.type === NOTIFICATION_TYPES.user;
 
   return (
     <NotificationContainer isOld={isOldNotification}>
       <ImageWrapper>
         <img alt="notification" src={isOldNotification ? warningIcon : redWarningIcon} />
       </ImageWrapper>
-      <div>
-        <Title>{note.title}</Title>
-        <Description>{note.description}</Description>
-      </div>
+      {isUserNotifications ? (
+        <div>
+          <Title>{`${note.courseName} is ${note.title}`}</Title>
+          <Description>{note.description}</Description>
+        </div>
+      ) : (
+        <div>
+          <Title>{note.userName}</Title>
+          <Description>{`${note.description} ${note.courseName}`}</Description>
+        </div>
+      )}
```

Comment:
```suggestion
      {isUserNotifications ? (
        <div>
          <Title>{`${courseName} is ${title}`}</Title>
          <Description>{description}</Description>
        </div>
      ) : (
        <div>
          <Title>{userName}</Title>
          <Description>{`${description} ${courseName}`}</Description>
        </div>
      )}
```

Reply:
and also maybe <div> can be replaced with empty tag in this case?

Reply:
No, our notification style is breaking down